dnf/yum/config.py

```diff
@@ -54,11 +56,7 @@ def __init__(self):
         self._add_option('exclude', self._get_option('excludepkgs'))
         self._add_option('config_file_path', PathOption("/etc/yum/yum.conf"))
```

Comment:
Sorry, I have found another problem with yum.conf. The yum conf is in Rhel and Fedore in /etc/yum.conf , but anyway we should keep dnf config_file_path due to:
 * dnf doesn't provide yum.conf
 * yum package is provided by yum.conf (yum-depricated)
 * maintains of two files is more difficult